### PR TITLE
Add nativeX route for ‘Industry Highlights’ section

### PR DESCRIPTION
**Pull request by jwade1327**
Screen Shot 2022-12-01 at 11 50 30 AM
Screen Shot 2022-12-01 at 11 56 56 AM
Screen Shot 2022-12-01 at 11 57 20 AM

### CODE REVIEW

**Comment by jwade1327** [comment]
[Add NativeX Ads to Industry Highlights Channels for the following sites (like we did for Sessions): https://www.aadmeetingnews.org https://isc.hub.heart.org https://www.auadailynews.org/](https://trello.com/c/Qkt8NgUo/404-add-nativex-ads-to-industry-highlights-channels-for-the-following-sites-like-we-did-for-sessions-https-wwwaadmeetingnewsorg-http)

**Review by Shinsina — CHANGES_REQUESTED** [review]
Ref: https://github.com/parameter1/ascend-media-websites/pull/381 These need to have an `industry-highlights` Native-X channel created in order for this to "work as intended"

File: sites/auadailynews.org/config/site.js [hunk 1]
```diff
@@ -54,4 +54,5 @@ module.exports = {
     enabled: true,
     cookieDomain: process.env.NODE_ENV === 'production' ? 'auadailynews.org' : '',
   },
+  nativeXHeroSections: ['industry-highlights'],
```
Comment (Shinsina) [inline sites/auadailynews.org/config/site.js:57]: I don't believe this needs to exist anywhere anymore, this was likely forgotten about when we switch it from just the hero block pulling Native-X ads to the entire section page template. Can you confirm?

Comment (jwade1327) [reply sites/auadailynews.org/config/site.js]: @Shinsina Can confirm this is not needed (I also removed it from sessions site config)

**Review by Shinsina — APPROVED** [review]

**Comment by Shinsina** [comment]
[Industry Highlights section change to all Native](https://trello.com/c/7wNagrTL/415-industry-highlights-section-change-to-all-native)

**Pull request merged**